public/scripts/power-user.js

```diff
@@ -1097,7 +1097,7 @@ export function formatInstructModePrompt(name, isImpersonate, promptBias, name1,
     }
 
     const separator = power_user.instruct.wrap ? '\n' : '';
-    let text = includeNames ? (separator + sequence + separator + `${name}:`) : (separator + sequence);
+    let text = includeNames ? (separator + sequence + separator + `${name}: `) : (separator + sequence);
```

Comment:
When adding a name and a colon, we should always add a space, too! Otherwise the model has to add the space on its own, and that can cause problems since spaces are tokens, too, affected by repetition penalty and samplers. However, even though I added the space here, it seems to get trimmed somewhere else, so there's a follow-up change required.

Reply:
Maybe it's a good thing if we limit the tokens to those which don't start with a leading space. Because those probably wouldn't be suitable tokens to start a new sentence with anyway.

But that's just my guess and would need further testing for confirmation. Alternatively, we could look at how Ooba and Kobold do it, and if they both do it the same way, it might be best to just do it that way as well.

Reply:
I've removed the space I added after the name because koboldcpp doesn't do that, either. More in another comment, but this conversation is thus resolved.